R/download_survey.R

```diff
@@ -106,28 +106,42 @@ download_survey <- function(survey, dir = NULL, sleep = 1) {
     zenodo_links <- zenodo_links[!duplicated(file_name)]
   }
 
-  if (is.null(dir)) {
-    dir <- tempdir()
+  dir <- dir %||% get_pkg_user_dir()
```

Comment:
would it make more sense to just have `dir = get_pkg_user_dir()` as default argument for transparency (and update the doc accordingly)?